### PR TITLE
Vendor npm assets

### PR DESCRIPTION
**Pull request by alex-ju**
This PR
- vendors all npm JavaScript files to allow deployment and ignores them when linting
- reverts configuration changes to 6ecd117

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,9 +20,11 @@
   },
   "homepage": "https://github.com/alphagov/content-publisher#readme",
   "dependencies": {
-    "@github/markdown-toolbar-element": "^0.1.0",
-    "@webcomponents/webcomponentsjs": "^2.0.4",
-    "nodelist-foreach-polyfill": "^1.2.0",
     "standard": "^11.0.1"
+  },
+  "standard": {
```
Comment (benthorner) [inline package.json]: Can we do a 'git checkout package.json 6ecd117' here?

**Comment by benthorner** [comment]
Similarly, I think we can checkout startup.sh as we don't need to install on startup anymore :-).

**Comment by alex-ju** [comment]
@benthorner reverted the files to 6ecd117

**Comment by benthorner** [comment]
We can also revert app.json ;-).

**Pull request merged**